helpers/errorCodes.js

```diff
@@ -316,6 +316,11 @@ module.exports = {
 		message: 'Could not upload application'
 	},
 
+	E1007: {
```

Comment:
Remember to add new Error codes to: https://github.com/Educado-App/resources/blob/main/ERRORS.md